Explorer/Assets/DCL/ApplicationVersionGuard/SemanticVersioningExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text.RegularExpressions;
+
+namespace DCL.ApplicationVersionGuard
+{
+    public static class SemanticVersioningExtensions
+    {
+        public static bool IsOlderThan(this string current, string latest) =>
+            current.ToSemanticVersion().IsOlderThan(latest.ToSemanticVersion());
+
+        private static bool IsOlderThan(this (int, int, int) current, (int, int, int) latest)
+        {
+            if (current.Item1 < latest.Item1) return true;
+            if (current.Item2 < latest.Item2) return true;
+            return current.Item3 < latest.Item3;
+        }
```

Comment:
Can we give each of the "Item{number}" a descriptive a name? I don't remember if it allows it, but I think you were able to set these names in the definition:
```
(int name1, int name2, int name2) current
```

(Correct me if I'm wrong)